### DISCUSSION TITLE
Lugares para Coworking

### DISCUSSION THREAD

**Discussion by Matheus-de-Souza** [frontendbr/forum#763] [Discussão]
Eu estava acompanhando a issue sobre [home office](https://github.com/frontendbr/forum/issues/733) e vi um [comentário muito legal](https://github.com/frontendbr/forum/issues/733#issuecomment-336019004) do @victorferraz sobre um site de uma empresa de coworking a [BeerCoffee](https://beerorcoffee.com), eu nem sabia, mas ela está aqui na minha cidade também (Novo Hamburgo/RS).

Eu queria saber se alguém conhece mais lugares de coworking pelo Brasil ou fora do Brasil também, me parece que essas informações podem ser útil a outros também.

**Comment by Zutzsa** [comment]
No shopping Via parque que fica na Barra da tijuca, Rio de Janeiro tem uma área para coworking.

Segue o link referente ao espaço de coworking:
http://viaparqueshopping.com.br/novidades-e-eventos/coworking.htm

**Comment by victorferraz** [comment]
@Matheus-de-Souza uma coisa legal também, é que a maioria dos coworkings são cadastrado no google places, eu vou no google maps peço para setar minha localização atual e depois busco os coworkings mais próximos de mim, sempre funciona :)

**Comment by Matheus-de-Souza** [comment]
Que legal isso @victorferraz, não sabia dessa ferramenta, achei mais uns dois espaços aqui perto

**Comment by willianjusten** [comment]
Eu busco não só Coworking, mas costumo usar o filtro `notebook friendly` enquanto estou viajando. Com isso sempre acabo achando uns cafés bem legais e ótimos para trabalhar. Em Praga eu descobri um lugar maravilhoso, que é o [Cafedu](http://www.cafedu.cz/english/), é um café que funciona 24h para que estudantes possam ficar lá e talz, foi um achado super legal!

Eu tava com o objetivo de fazer um projeto aberto sobre isso, já que eu tenho trabalhado remoto por esse tempo todo, mas acabou que mil coisas vieram aparecendo e eu não consegui dar um início...
